RELEASING/Dockerfile.from_local_tarball

```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y apt-transport-https apt-utils
 # Install superset dependencies
 # https://superset.apache.org/docs/installation/installing-superset-from-scratch
 RUN apt-get install -y build-essential libssl-dev \
-    libffi-dev python3-dev libsasl2-dev libldap2-dev libxi-dev
+    libffi-dev python3-dev libsasl2-dev libldap2-dev libxi-dev chromium
```

Comment:
could be more defensive and add an extra line with:
```
RUN if [ $(arch) == "aarch64" ]; then apt-get install chromium; fi
```

But would add another layer to the container and it should be fine on non M1 also